### PR TITLE
Master import export command

### PR DESCRIPTION
**Pull request by s7design**
Hi @UdoBretz 

Here are added Console Command for Import Export package.
As you can see,  Import and Export are splitted in two commands. At first I had done in one, and then I splitted it. I have this version on local. If you prefer such solution please let me know and I will be able to provide only one Console command 'Admin::ITSM::ImportExport' and a test for it.

I added a test file scripts/test/sample/ImportExport/TemplateExport.csv which is used in Import.t Command test. 

Regards
Zoran

### CODE REVIEW

**Comment by UdoBretz** [comment]
Hi Zoran,

thanks a lot! I like your solution of splitting the import and the export command.

Best regards,
Udo

File: Kernel/System/Console/Command/Admin/ITSM/ImportExport/Export.pm [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    );
+
+    if ( !$Result ) {
+        $Self->PrintError("Error occurred. Export impossible! See Syslog for details.\n");
+        return $Self->ExitCodeError();
+    }
+
+    $Self->Print( "<green>" . ( '-' x 69 ) . "</green>\n" );
+    $Self->Print("<green>Success: $Result->{Success}</green>\n");
+    $Self->Print("<green>Failed : $Result->{Failed}</green>\n");
```
Comment (mgruner) [inline Kernel/System/Console/Command/Admin/ITSM/ImportExport/Export.pm]: <green>? For errors I'd recommend PrintError.

File: Kernel/System/Console/Command/Admin/ITSM/ImportExport/Import.pm [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+    if ( !$Result ) {
+        $Self->PrintError("\nError occurred. Import impossible! See the OTRS log for details.\n");
+        return $Self->ExitCodeError();
+    }
+
+    # print result
+    $Self->Print("\n<green>Import of $Result->{Counter} $Result->{Object} records:</green>\n");
+    $Self->Print( "<green>" . ( '-' x 69 ) . "</green>\n" );
+    $Self->Print("<green>$Result->{Failed} failed, $Result->{Success} succeeded</green>\n");
```
Comment (mgruner) [inline Kernel/System/Console/Command/Admin/ITSM/ImportExport/Import.pm]: Similar.

File: Kernel/System/Console/Command/Admin/ITSM/ImportExport/Export.pm (hunk 1 above)
Comment (s7design) [inline Kernel/System/Console/Command/Admin/ITSM/ImportExport/Export.pm]: Yes of course it is my mistake. I will fix it soon
Regards
Zoran

**Pull request closed without merging**